_data/redirects.yaml

```diff
@@ -103,6 +77,7 @@
   destination: https://www.youtube.com/watch?v=WK5tCf7m-ZM
 - source: /soon
   destination: /calendar
+<<<<<<< HEAD
```

Comment:
You've left in the conflict markers here and at the end of the file

Reply:
If everything else is good you can just cut that out when merging